x-pack/plugins/security_solution/public/app/home/template_wrapper/index.tsx

```diff
@@ -25,6 +25,7 @@ import { useShowTimeline } from '../../../common/utils/timeline/use_show_timelin
 import { gutterTimeline } from '../../../common/lib/helpers';
 import { useKibana } from '../../../common/lib/kibana';
 import { useShowPagesWithEmptyView } from '../../../common/utils/empty_view/use_show_pages_with_empty_view';
+import { useIsPolicySettingsBarVisible } from '../../../management/pages/policy/view/policy_hooks';
```

Comment:
This is very specific to the Policy Details view, but the issue (I think) would visible anytime the Eui's Bottom bar component is visible. Any way to perhaps detect that instead of the implementation by the policy details?

Reply:
Slack conversation summary:
Parkiino - Besides adding any paths that use the bottom bar to an array, I'm not too sure how we would check for the presence of a bottom bar
Paul-Tavares - The only other way would be to query the Dom for the bar using it css class name by doing `document.body.querySelector()`. Some concerns:
The fix is very specific to the Policy Details page, when really it should be more about detecting if the sticky bar is being used. The possible (future) issue there is that if we change the policy details to now use the sticky bar, then we’re leaving code around that still assumes the page uses it. Also, if another area of the app starts to use the Sticky bottom bar, then they will also have the problem and we’ll get this bug reported **again**